lib/app/utils/terminology.rb

```diff
@@ -28,6 +28,8 @@ class Terminology
 
     @known_valuesets = {}
 
+    @@loaded_validators = {}
```

Comment:
why class variable instead of instance variable?

I think you should use an instance variable here similar to `@known_valuesets`

Reply:
In that case I'd recommend using a class instance variable rather than a class variable. If you refer to an instance variable within a class method, you get a class instance variable:
https://github.com/onc-healthit/inferno/blob/development/lib/app/sequence_base.rb#L311

They're basically like class variables without the weirdness around inheritance.

Reply:
Thats sort of what is being done with `@known_valuesets`, minus the method that exposes it.

Reply:
Ugh, given the current prevalance of class variables here I wonder whether it's really worth changing it. If we really care about avoiding class variables, I think the thing to probably do is wrap pretty much the entire class body in a `class << self` block and replace all the  class variables with instance variables. Then you could use attr_reader/writers, too. Don't know that it makes sense to undertake that as part of this ticket, though.

Reply:
I wouldn't remove the other ones, just make this one a class instance variable.

This other [PR](https://github.com/onc-healthit/inferno/pull/234) removes them all from the class already.  I just don't want to add another.

Reply:
unless you don't think the class instance variables are any better.

Reply:
Since the other PR removes them, it makes sense for this to be a class instance variable.

Reply:
Sure, that works for me.